src/renderer/components/GeneralSettings/index.tsx

```diff
@@ -34,8 +34,24 @@ function InstallPathSettingItem(props: { path: string, setPath: (path: string) =
     );
 }
 
+const DisableWarningSettingItem = (props: {disableWarning: boolean, setDisableWarning: CallableFunction}) => {
+    const handleClick = () => {
+        const newState = !props.disableWarning;
+        props.setDisableWarning(newState);
+        settings.set('mainSettings.disableExperimentalWarning', newState);
+    };
+
+    return (
+        <SettingsItem>
+            <SettingItemName>Disable Version Warnings</SettingItemName>
+            <SettingItemContent onClick={handleClick}>{props.disableWarning ? 'Yes' : 'No'}</SettingItemContent>
+        </SettingsItem>
+    );
+};
+
 function index(): JSX.Element {
     const [installPath, setInstallPath] = useState<string>(settings.get('mainSettings.msfsPackagePath') as string);
+    const [disableWarning, setDisableWarning] = useState<boolean>(settings.get('mainSettings.disableExperimentalWarning') as boolean);
 
     const handleReset = async () => {
         settings.clear();
```

Comment:
![grafik](https://user-images.githubusercontent.com/50082450/109946517-9c2e6b00-7cd8-11eb-96da-d0785a6b67af.png)
While waiting for GH access, I'm gonna do it this way... This fixes the state not being displayed correctly after pressing reset

Reply:
settings.reset handles it automatically, I could change the state though